scripts/test_generated_output.py

```diff
@@ -0,0 +1,173 @@
+            else:
+                try:
+                    spec.on_attestation(store, attestation, is_from_block=False)
+                    assert False
+                except AssertionError:
+                    pass
+        elif 'attester_slashing' in step:
+            slashing_id = step['attester_slashing']
+            valid = step.get('valid', True)
+            recovery = step.get('recovery', False)
+            assert valid
+            slashing = slashings[slashing_id]
+            spec.on_attester_slashing(store, slashing)
+        elif 'checks' in step:
+            checks = step['checks']
+            for check, value in checks.items():
+                if check == 'time':
+                    expected_time = value
+                    assert store.time == expected_time
+                elif check == 'head':
+                    assert str(spec.get_head(store)) == value['root']
+                elif check == 'proposer_boost_root':
+                    assert str(store.proposer_boost_root) == str(value)
+                elif check == 'justified_checkpoint':
+                    checkpoint = store.justified_checkpoint
+                    assert checkpoint.epoch == value['epoch']
+                    assert str(checkpoint.root) == str(value['root'])
+                elif check == 'finalized_checkpoint':
+                    checkpoint = store.finalized_checkpoint
+                    assert checkpoint.epoch == value['epoch']
+                    assert str(checkpoint.root) == str(value['root'])
+                elif check == 'viable_for_head_roots_and_weights':
+                    filtered_block_roots = spec.get_filtered_block_tree(store).keys()
+                    leaves_viable_for_head = [root for root in filtered_block_roots
+                                            if not any(c for c in filtered_block_roots if store.blocks[c].parent_root == root)]
+                    viable_for_head_roots_and_weights = {
+                        str(viable_for_head_root): int(spec.get_weight(store, viable_for_head_root))
+                        for viable_for_head_root in leaves_viable_for_head
+                    }
+                    expected = { kv['root']: kv['weight'] for kv in value}
+                    assert expected == viable_for_head_roots_and_weights
+                else:
+                    assert False
+        else:
+            assert False
+
+
+def gather_tests(tests_dir) -> Iterable[TestInfo]:
+    for preset in [p.name for p in Path(tests_dir).glob('*') if p.name in spec_targets]:
+        for fork in [f.name for f in (Path(tests_dir) / preset).glob('*') if f.name in spec_targets[preset]]:
+            print(f'{preset}/{fork}')
+            for test_dir in sorted([td for td in (Path(tests_dir) / preset / fork).glob('*/*/*/*')]):
+                yield TestInfo(preset, fork, test_dir)
+
+
+def runt_tests_parallel(tests_dir, num_proc=settings.NUM_PROCESS):
```

Comment:
Pretty sure this is a typo.
```suggestion
def run_tests_parallel(tests_dir, num_proc=settings.NUM_PROCESS):
```